### ISSUE TITLE
Allow logic for questions

### ISSUE THREAD

**Issue by suzannehillman** [simonv3/quick-survey#50]
In many cases, how one answers a question will affect the rest of the questions that one should be asked. Because of this, it would be wonderful if logic existed for the questions. 

For example, if a question asks if someone has a car, there may be followup questions for people who say they do (what kind of car, what do you use it for, etc). 

As it is, all questions have to be asked, regardless of if they are relevant to the particular person filling out your form. This can result in people not actually completing the form due to annoyance, overload, or time constraints.

**Comment by simonv3** [comment]
Yeah, this is a high level architecture change that I'm not sure how I should implement data-wise. Been idling on it for a while.
